client/src/components/feeds/post/comments/CommentSection.tsx

```diff
@@ -99,11 +97,7 @@ function CommentSection(props: CommentSectionProps) {
     setComments(threads);
   }, [props.comments]);
 
-  useEffect(() => {
```

Comment:
As a result of this change, the optimistic update when submitting a top level comment no longer appears to work.

Demo: https://streamable.com/w5wmn0